### PR TITLE
Turtle-like movement functions

### PR DESCRIPTION
**Pull request by JensKanstrupLarsen**

### CODE REVIEW

File: agents/model.py [hunk 1]
```diff
@@ -115,11 +115,26 @@ def point_towards(self, other_x, other_y):
...
         self.__post_move()
 
+    def backward(self, distance=None):
+        if distance == None:
+            distance = self.speed
+        self.x -= math.cos(math.radians(self.direction)) * distance
+        self.y -= math.sin(math.radians(self.direction)) * distance
+        self.__post_move()
+
+    def agent.left(self, degrees):
```
Comment (dybber) [inline agents/model.py:132]: shouldn't it just be `def left(self, degrees)`?

File: agents/model.py [hunk 2]
```diff
@@ -115,11 +115,26 @@ def point_towards(self, other_x, other_y):
...
+        self.direction += degrees
```
Comment (dybber) [inline agents/model.py:133]: shouldn't it just be `def left(self, degrees)`?

File: agents/model.py [hunk 3]
```diff
@@ -115,11 +115,26 @@ def point_towards(self, other_x, other_y):
...
+
+    def agent.right(self, degrees):
```
Comment (dybber) [inline agents/model.py:135]: Same here

**Review by dybber — CHANGES_REQUESTED** [review]

File: agents/model.py (hunk 2 above)
Comment (JensKanstrupLarsen) [reply agents/model.py:133]: Oh yeah.

**Review by dybber — APPROVED** [review]

**Pull request merged**